.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2023 Linaro Ltd.
+#
+name: "Builds"
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run periodically to check that it still compiles
+    - cron: '13 13 * * 1'
+  workflow_dispatch:
+
+jobs:
+  job:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - debian:testing
+          - debian:bookworm
+            #- debian:bullseye
+            #- debian:buster
+          - ubuntu:lunar
+          - ubuntu:jammy
+            #- ubuntu:focal
+            #- ubuntu:bionic
+            #- ubuntu:xenial
+
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+
+      - name: Install meson
+        run: |
+          apt update
+          apt -y install --no-install-recommends meson build-essential
```

Comment:
The GitHub actions container might already provide this by default?

Reply:
No, it doesn't. I had to install those manually.

Reply:
I mean, I think the default container had some tools. But after switching to `debian:foo` and `ubuntu:bar` I had to install the tools.

Reply:
Right, yeah, you're not using the GitHub container but some debian/ubuntu base for some reason.